### DISCUSSION TITLE
renderin existing pdf file

### DISCUSSION THREAD

**Discussion by kair0span** [diegomura/react-pdf#2760] [General]
It is possible to render existing pdf file with the plugin without creating a new one?

**Comment by diegomura** [comment]
Hi! No, this is not possible. react-pdf only generates new PDF documents from React components. It cannot open, parse or display an existing PDF file.

If what you need is to display an existing PDF in a React app, take a look at [react-pdf by wojtekmaj](https://github.com/wojtekmaj/react-pdf), which is a different library made exactly for that.

Closing this one since it is out of scope for this library. Thanks!
